traefikee/values.yaml

```diff
@@ -120,8 +126,12 @@ controller:
       providers:
         kubernetesIngress:
           allowEmptyServices: true
+          # let it as is or specify you custom ingressClass name if any
+          ingressClass:
         kubernetesCRD:
           allowEmptyServices: true
+          # let it as is or specify you custom ingressClass name if any
```

Comment:
```suggestion
```

Reply:
fix doc wording for KubernetesCRD provider in static